### PR TITLE
Update Terraform Version

### PR DESCRIPTION
**Pull request by aishwaryabk**
Signed-off-by: Aishwarya Kamat <aishwarya.kamat@ibm.com>

### CODE REVIEW

File: modules/1_prepare/versions.tf [hunk 1]
```diff
@@ -29,5 +29,5 @@ terraform {
       version = "~> 2.1"
     }
   }
-  required_version = "~> 0.13.0"
+  required_version = "~> 1.0.8"
```
Comment (yussufsh) [inline modules/1_prepare/versions.tf:32]: Why 1.0.8? If I am not wrong there is no major changes and it should work with any version greater than 1.0.0.
Also, we should use `>=` to auto support `1.1.0` which is under beta currently.

File: docs/automation_host_prereqs.md [hunk 2]
```diff
@@ -21,10 +21,10 @@ Install the following packages on the automation host. Select the appropriate in
 
 ### Terraform
 
-**Terraform >= 0.13.0**: Please refer to the [link](https://learn.hashicorp.com/terraform/getting-started/install.html) for instructions on installing Terraform. For validating the version run `terraform version` command after install.
+**Terraform >= 1.0.0**: Please refer to the [link](https://learn.hashicorp.com/terraform/getting-started/install.html) for instructions on installing Terraform. For validating the version run `terraform version` command after install.
 
 Install Terraform and providers for Power environment:
-1. Download the Terraform binary version 0.13.6 from https://www.power-devops.com/terraform and install it to /usr/local/bin.
+1. Download the Terraform binary version 1.0.8 from https://www.power-devops.com/terraform and install it to /usr/local/bin.
```
Comment (yussufsh) [inline docs/automation_host_prereqs.md:27]: `Download and install the latest (>= 1.0.0) Terraform binary for Linux/ppc64le from https://www.power-devops.com/terraform`

**Review by yussufsh — CHANGES_REQUESTED** [review]

File: modules/1_prepare/versions.tf (hunk 1 above)
Comment (aishwaryabk) [reply modules/1_prepare/versions.tf]: Thanks. Updated the change.

**Comment by yussufsh** [comment]
Let me raise the PR for the prow validate job to use the latest terraform.

**Comment by cs-zhang** [comment]
Do we need to define TF version in all sub-modules? Just use the root one maybe enough, that make the change easier.

**Comment by yussufsh** [comment]
@cs-zhang @aishwaryabk any thoughts on how we can upgrade to >v1.0 without affecting the openshit-install-power project?
Should be create tags and use it in openshit-install-power? Because once you upgrade then older versions of the script will not be able to work when they will download the new code.

**Comment by cs-zhang** [comment]
Yes, after cherry-pick the change to all supported release branches, we can to update openshift-instakk-power script to use TF 1.x.

**Comment by aishwaryabk** [comment]
There is a bug encountered on the TF v1.1.0; so, limiting the version to 1.0.x

**Comment by yussufsh** [comment]
@aishwaryabk @cs-zhang I got an idea that I have not tried.
We should allow both `~> 0.13.0` and `>= 1.0.0` so that existing openshift-install-powervs script does not fail. Once the dusts settle we can remove `~> 0.13.0`.

**Comment by cs-zhang** [comment]
The PR 387 fixed the scp error, that fixed TF 1.1.x problem, so yes, we can use `>= 0.13.0` to allow all TF version after 0.13.0.

**Comment by aishwaryabk** [comment]
@yussufsh @cs-zhang Thanks for the suggestions. I will update the files to support all versions `>= 0.13.0 `

**Review by yussufsh — APPROVED** [review]
/lgtm

**Comment by sajauddin** [comment]
Is there any specific reason for openshift-install-power is not being updated to TF 1.0.x or later

**Comment by yussufsh** [comment]
There was this dependency issue with folks still using older version of script. Once we merge this change the script can start using the latest version of Terraform and old versions as well will work. 
After a while we should only support Terraform 1.0 and above.

**Review by cs-zhang — APPROVED** [review]
/lgtm

**Pull request merged**